src/librustdoc/doctest/make.rs

```diff
@@ -520,6 +539,8 @@ fn handle_attr(mod_attr_pending: &mut String, source_info: &mut SourceInfo, edit
         mod_attr_pending.clear();
     } else if mod_attr_pending.ends_with('\\') {
         mod_attr_pending.push('n');
```

Comment:
If I understand this correctly, these lines aren't really needed any more. They would have turned

```rust
#[doc="first\
second"]
```

into

```
#[doc="first\nsecond"]
```

But that only works with double-quoted strings, not raw strings, and is unnecessary now that newlines are preserved here.

```suggestion
```

Reply:
Adding a test just in case.